report/main.qmd

```diff
@@ -0,0 +1,139 @@
+
+[^2]: For example, day 199 in the C site has more zero-calorie and sugary beverages sold than total items. Also, days 50 to 53, 107, 184 and 196 in the same site have decimals.
+
+As sites B and C have a short pre-intervention period, it is necessary to compensate for the missing information. This can be done by combining the follow-up and pre-intervention categories into a new category that will serve as the baseline for the study. Lastly, we are not interested in the washout periods that are used to reduce the effects of previous interventions. Hence, this data can be ignored.
+
+## ANOVA/ANCOVA
+
+Analysis of Variance (ANOVA) is a statistical method used to compare the means of three or more samples, determining if at least one sample mean significantly differs from the others. It's instrumental in experiments aimed at evaluating the effectiveness of different interventions. In this study, it is used to encourage the selection of zero-calorie beverages over sugary options in hospital settings. By examining the variance within and between intervention groups, ANOVA facilitates understanding whether observed differences in beverage consumption are attributable to the interventions or occur by chance.
+
+However, ANOVA's limitation lies in its inability to account for external variables that could influence outcomes, such as baseline consumption patterns or hospital site characteristics. Despite this, ANOVA remains valuable for initial analysis, with its limitations highlighting the need for a comprehensive approach, possibly incorporating Analysis of Covariance (ANCOVA), to fully understand the interventions’ impacts on beverage selection.
+
+## Linear Mixed-Effect Model
+
+Linear Mixed-Effects (LME) model is a statistical model that accounts for fixed effects, common trends that are present at all levels of the data, as well as random effects, correlation within groups and variation between groups. The inclusion of random effects makes LME especially adept at handling hierarchical and longitudinal data, where observations are grouped into different levels. In these models, we assume that the errors (or the differences between the predicted value and the actual value) are random and follow a normal distribution with an average of zero. 
```

Comment:
I understand that you would like to make the explanation of fixed/random effect intuitive, but "correlation within groups and variation between groups" is not accurate enough to illustrate the ideas. At least the intuitive interpretations should be given under the context of linear regression.